sgl-router/src/routers/grpc/harmony/responses.rs

```diff
@@ -1418,6 +1467,38 @@ pub fn convert_mcp_tools_to_response_tools(mcp_tools: &[mcp::Tool]) -> Vec<Respo
         .collect()
 }
 
+/// Convert tools map to ResponseTools with formatted names (server_label__tool_name)
+///
+/// This function formats tool names as `server_label__tool_name` to prevent name collisions
+/// across multiple MCP servers and enable server-specific tool routing.
+fn convert_mcp_tools_map_to_response_tools(
```

Comment:
where is `convert_mcp_tools_to_response_tools` used then? why do we need two of these now?